### DISCUSSION TITLE
Support for Postgres LISTEN/NOTIFY

### DISCUSSION THREAD

**Discussion by LucaCappelletti94** [diesel-rs/diesel#3955] [Q&A] [answered]
Hi all, what is the current support in Diesel for LISTEN/NOTIFY?

**Comment by weiznich** [comment, marked as answer]
There is currently no support for `LISTEN/NOTIFY` by the built-in `PgConnection` type. I cannot speak for potential existing third party connection implementations.

**Comment by thkoch2001** [comment]
Postgres Notifications support has been merged:
https://github.com/diesel-rs/diesel/pull/4420

**Reply by LucaCappelletti94** [reply]
Thank you for letting me know!
